pandas/tests/test_expressions.py

```diff
@@ -380,3 +380,48 @@ def test_frame_series_axis(self, axis, arith):
 
         result = op_func(other, axis=axis)
         tm.assert_frame_equal(expected, result)
+
+    @pytest.mark.parametrize(
+        "op",
+        [
+            "__mod__",
+            pytest.param("__rmod__", marks=pytest.mark.xfail(reason="GH-36552")),
+            "__floordiv__",
+            "__rfloordiv__",
+        ],
+    )
+    @pytest.mark.parametrize(
+        "box, tester",
+        [
+            (DataFrame, tm.assert_frame_equal),
+            (Series, tm.assert_series_equal),
```

Comment:
we don't need to usually add the tester as assert_equal handles this

Reply:
good point. a bit belt and braces. will change so we re-run ci b4 merge.